### PR TITLE
Cy/add monitor script

### PR DESCRIPTION
**Pull request by cy-dream**

### CODE REVIEW

File: config.yml.sample [hunk 1]
```diff
@@ -0,0 +1,16 @@
+engineer_recipients:
+  - 'yuechao.li@wehome.io'
+  - 'jerry.an@wehome.io'
+  - 'wenhang@wehome.io'
+  - 'haorui@wehome.io'
+
```
Comment (hethune) [inline config.yml.sample]: +vivian

File: run.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+''' 
+python-tail example.
```
Comment (hethune) [inline run.py]: change the description

File: run.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import yaml
+import requests
+
+with open('config.yml') as f:
+  config = yaml.load(f)
+
+def return_line(msg):
+  ''' Prints received text '''
+  if 'Failed' in msg or 'Exception' in msg:
+    if len(msg) > 2000:
```
Comment (hethune) [inline run.py]: make `2000` a constant

File: run.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import tail
+import yaml
+import requests
+
+with open('config.yml') as f:
+  config = yaml.load(f)
+
+def return_line(msg):
+  ''' Prints received text '''
+  if 'Failed' in msg or 'Exception' in msg:
```
Comment (hethune) [inline run.py]: define a constant e.g
`STOP_WORDS = ['Failed', 'Exception']`

File: run.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    if len(msg) > 2000:
+      msg = msg[0:2000]
+    testFlask(msg, config['engineer_recipients'])
+   
+def send_msg(msg, recipients):
+  values = {'recipients' : recipients,
+          'msg' : msg
+  }
+  url = 'http://localhost:5000/message'
```
Comment (hethune) [inline run.py]: make url a config

File: run.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+with open('config.yml') as f:
+  config = yaml.load(f)
+
+def return_line(msg):
+  ''' Prints received text '''
+  if 'Failed' in msg or 'Exception' in msg:
+    if len(msg) > 2000:
+      msg = msg[0:2000]
+    testFlask(msg, config['engineer_recipients'])
```
Comment (hethune) [inline run.py]: what is `testFlask`?

**Comment by cy-dream** [comment]
@hethune

File: config.yml.sample [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    stop_words:
+      - 'Failed'
+      - 'Exception'
+    recipients:
+      - 'yuechao.li@wehome.io'
+      - 'jerry.an@wehome.io'
+      - 'wenhang@wehome.io'
+      - 'haorui@wehome.io'
+      - 'vivian.zhang@wehome.io'
+  operation_group:
```
Comment (hethune) [inline config.yml.sample]: `operation_groups` would be a better name

File: config.yml.sample [hunk 8]
```diff
@@ -0,0 +1,24 @@
+sending_pattern:
```
Comment (hethune) [inline config.yml.sample]: just make `sending_patterns` an array
```
sending_pattern:
  - stop_words: "xxxx"
    recipients:  xxxxx
    group_name: "xxx:
  - stop_words: "xxxx"
    recipients:  xxxxx
    group_name: "xxx:
```

File: run.py [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+import tail
+import yaml
+import requests
+
+with open('config.yml') as f:
+  config = yaml.load(f)
+
+def return_line(msg):
+  ''' Prints received text '''
+  for groupName in config['sending_pattern']:
```
Comment (hethune) [inline run.py]: with the changes above, just need to do 
```
for pattern in config['sending_patterns']:
   xxxx
```

**Comment by cy-dream** [comment]
@hethune  修改配置文件

**Comment by hethune** [comment]
LGTM
Squash commits before merging

**Pull request merged**